addons/account/migrations/9.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,935 @@
+---Fields in module 'account'---
+account      / account.account          / active (boolean)              : DEL 
+account      / account.account          / child_consol_ids (many2many)  : DEL relation: account.account
+account      / account.account          / child_parent_ids (one2many)   : DEL relation: account.account
+account      / account.account          / currency_mode (selection)     : DEL required: required, selection_keys: ['average', 'current'], req_default: current
+account      / account.account          / deprecated (boolean)          : NEW 
+# TODO: in post-migration, if the active field is False, set it to Trueg
+account      / account.account          / financial_report_ids (many2many): DEL relation: account.financial.report
+account      / account.account          / last_time_entries_checked (datetime): NEW 
+# The code in account module that uses this field is commented... So seems that it has no purpose (and no motivation to build a migration script)
+account      / account.account          / parent_id (many2one)          : DEL relation: account.account
+account      / account.account          / parent_left (integer)         : DEL 
+account      / account.account          / parent_right (integer)        : DEL 
+account      / account.account          / shortcut (char)               : DEL 
+account      / account.account          / tag_ids (many2many)           : NEW relation: account.account.tag
+# NOTHING TO DO: The migration of tags should be provided by the l10n account modules 
```

Comment:
@pedrobaeza, but the module account now does not add any COA at all, not even the generic. The generic is provided by module l10n_generic_coa.

Now, a generic method that converts parent accounts into tags would be useful for one or more localizations, so I understand the justification for a generic method, that will not be called in the migration of account, but during the migration of the other modules.